portfolio/src/main/webapp/index.html

```diff
@@ -34,23 +34,32 @@ <h3> Projects </h3>
       <h4> Fun fact about me: </h4>
       <p> I love In-N-Out! </p>
       <img src="/images/innout.jpg" width="300" height="300"/>
+      <br/>
+      <br/>
+
+      <!-- Fetch response from servlet -->
+      <button onclick = "fetchRandom();">
+      Display random variable
```

Comment:
Nit: Tab this line over to be nested in the button

Reply:
Done